src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk

```diff
@@ -27,9 +27,9 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
-      { defendantName: claim.claimData.defendant.name, totalAmountTillToday: claim.totalAmountTillToday | numeral,
-        moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+  <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
```

Comment:
this breaks the journey if you don't do a part admit now. I pulled your branch and when I did a paid in full after settlement agreement reached it shows £0. you need to add logic to show the different amount depending on the state of the claim

Reply:
Oh dear will take a look asap